src/components/common/modal/generic-modal.styles.tsx

```diff
@@ -11,6 +11,15 @@ export const ModalBox = styled(Modal.Box)`
     max-height: none;
 
     margin: auto;
+
+    [data-testid="close-button"] {
+        padding: 1rem;
+    }
+
+    svg {
+        width: 2rem;
+        height: 2rem;
+    }
```

Comment:
I presume these are to match the close button in figma, may I trouble you to check with UX if we can use the default size and padding from the DS?

Reply:
hi checked with karling alrdy she said we can stick with the DS styling for the cross button